.github/workflows/ci.yml

```diff
@@ -18,30 +18,36 @@ jobs:
             - name: Setup Python
               uses: actions/setup-python@v2
               with:
-                  python-version: 3.8
+                  python-version: '3.9'
 
             - uses: pre-commit/action@v2.0.0
 
     test-app:
 
         needs: [pre-commit]
 
-        runs-on: ubuntu-latest
-        timeout-minutes: 10
-
         strategy:
             matrix:
-                tag: [stable]
-                browser: [chrome, firefox]
+                tag: [edge]
+                browser: [Chrome, Firefox]
             fail-fast: false
 
+        runs-on: ubuntu-latest
+        timeout-minutes: 10
+
         steps:
 
             - name: Check out app
               uses: actions/checkout@v2
 
-            - name: Test app
-              uses: aiidalab/aiidalab-test-app-action@v2
+            - name: Setup Python
+              uses: actions/setup-python@v2
               with:
-                  image: aiidalab/aiidalab-docker-stack:${{ matrix.tag }}
-                  browser: ${{ matrix.browser }}
+                  python-version: '3.9'
```

Comment:
Maybe the Python version should be defined globally since it is at two places now.